modules/java/generator/src/java/org/opencv/utils/Converters.java

```diff
@@ -770,4 +772,41 @@ public static void Mat_to_vector_vector_char(Mat m, List<List<Byte>> llb) {
         }
         mats.clear();
     }
+
+    public static Mat vector_RotatedRect_to_Mat(List<RotatedRect> rs) {
+        Mat res;
+        int count = (rs != null) ? rs.size() : 0;
+        if (count > 0) {
+            res = new Mat(count, 1, CvType.CV_32FC5);
```

Comment:
Can we use `CvType.CV_32FC(5)` directly instead of adding of new const?

Reply:
sure, that was ugly, anyway ;)